examples/typescript/your_fungible_asset.ts

```diff
@@ -0,0 +1,227 @@
+/* eslint-disable no-console */
+/* eslint-disable max-len */
+
+import {
+  Account,
+  AccountAddress,
+  Aptos,
+  AptosConfig,
+  InputViewRequestData,
+  Network,
+  NetworkToNetworkName,
+} from "@aptos-labs/ts-sdk";
+import { compilePackage, getPackageBytesToPublish } from "./utils";
+/**
+ * This example demonstrate how one can compile, deploy, and mint its own fungible asset (FA)
+ * It uses the fa_coin.move module that can be found in the move folder
+ *
+ * Before running this example, we should compile the package locally:
+ * 1. Acquire the Aptos CLI, see https://aptos.dev/cli-tools/aptos-cli/use-cli/install-aptos-cli
+ * 2. cd `~/aptos-ts-sdk/examples/typescript`
+ * 3. Run `pnpm run your_coin`
+ */
+
+// Setup the client
+const APTOS_NETWORK: Network = NetworkToNetworkName[process.env.APTOS_NETWORK] || Network.TESTNET;
```

Comment:
Interesting, I used `DEVNET` for all of the others

Reply:
my bad